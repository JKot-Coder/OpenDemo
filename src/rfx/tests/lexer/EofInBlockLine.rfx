// :RUN_RFXC -L %STD_OUTPUT% %INPUT% 

/* EOF in block line