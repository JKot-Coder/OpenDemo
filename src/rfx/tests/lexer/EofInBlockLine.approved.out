LexerOutput:[
	{"Type":"line comment", "Content":"\/\/ -O LEXER", "Line":1, "Column":1},
	{"Type":"end of line", "Content":"\r\n", "Line":1, "Column":12},
	{"Type":"end of line", "Content":"\r\n", "Line":2, "Column":1},
	{"Type":"block comment", "Content":"\/* EOF in block line", "Line":3, "Column":1},
	{"Type":"end of file", "Content":"", "Line":3, "Column":21},
]
Diagnostic:[
	..\src\rfx\tests\lexer\EofInBlockLine.rfx(3): error 10006: end of file in block comment
	/* EOF in block line
	                    ^
]
